contract/contracts/PixelDevsUkraineDonation.sol

```diff
@@ -42,10 +44,11 @@ contract PixelDevsUkraineDonation is
     constructor() ERC721("PixelDevsUkraineDonation", "PXLDEV-UKRAINE") {
         _grantRole(DEFAULT_ADMIN_ROLE, msg.sender);
         _grantRole(WITHDRAW_ROLE, msg.sender);
+        setRoyalties(1000); // set royalties to 10%.
```

Comment:
This is not 10%, this is 1%

Change value into `10000` then we're good!

Reply:
I see thanks for pointing it out @itstargetconfirmed I may have confused it together with OpenSea's royalties. Because on OpenSea they only allow a maximum of 10% for royalties so I was also assuming it also applied the same thing for EIP2981 implementation.

But to be very sure, can you create unit tests for these and confirm it? Just call `royaltyInfo` method to calculate it the royalty amount

Reply:
@carlomigueldy Hi, I included the test cases with the initial pull request. I think we're good to go. Thank you for the feedback. I appreciate it.